build.sh

```diff
@@ -240,8 +245,15 @@ echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +
 echo "+               R U N   N E S T   T E S T S U I T E                           +"
 echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
 echo "MSGBLD0290: Running make installcheck."
-export PYTHONPATH=$HOME/.cache/csa.install/lib/python2.7/site-packages:$PYTHONPATH
-export LD_LIBRARY_PATH=$HOME/.cache/csa.install/lib:$LD_LIBRARY_PATH
+if [ "$TRAVIS_PYTHON_VERSION" == "2.7.13" ]; then
+    export PYTHONPATH=$HOME/.cache/csa.install/lib/python2.7/site-packages:$PYTHONPATH
+    export LD_LIBRARY_PATH=$HOME/.cache/csa.install/lib:$LD_LIBRARY_PATH
+elif [ "$TRAVIS_PYTHON_VERSION" == "3.4.4" ]; then
+    ls /usr/lib/x86_64-linux-gnu/
```

Comment:
I assume this line was for debugging. Please remove it.

Reply:
Done!